src/WingetCreateCLI/Program.cs

```diff
@@ -61,7 +65,13 @@ private static async Task<int> Main(string[] args)
                 Logger.Error(ex.ToString());
                 return 1;
             }
-        }
+        }
+
+        private static Type[] LoadVerbs()

```

Comment:
"Load" to me implies more than what this is doing, I believe Get would be a better prefix.

Reply:
Changed to GetVerbs()